lib/cc/pull_requests.rb

```diff
@@ -113,6 +134,10 @@ def setup_http
     raise NotImplementedError
   end
 
+  def welcome_comment_implemented?
+    raise NotImplementedError
```

Comment:
I think it would be fine to have this be `false` in the root class rather than forcing every class to specify.